src/fastapi_zitadel_auth/testing/fixtures.py

```diff
@@ -0,0 +1,136 @@
+"""
+Pytest fixtures for testing with fastapi-zitadel-auth
+
+To use these fixtures in your tests, add this to your conftest.py:
+
+    from fastapi_zitadel_auth.testing.fixtures import *
+"""
+
+from typing import Iterator, Any
+
+import httpx
+import pytest
+import respx
+
+from .utils import (
+    create_openid_keys,
+    openid_config_url,
+    openid_configuration,
+    keys_url,
+    MockZitadelAuth,
+    ZITADEL_CLIENT_ID,
+    ZITADEL_ISSUER,
+    ZITADEL_PROJECT_ID,
+)
+
+
+@pytest.fixture
+def mock_zitadel_auth() -> MockZitadelAuth:
+    """
+    Fixture providing a mock ZitadelAuth instance
+    
+    Returns a MockZitadelAuth that bypasses actual Zitadel validation.
+    Useful for testing application logic without network calls.
+    """
+    return MockZitadelAuth(
+        issuer_url=ZITADEL_ISSUER,
+        app_client_id=ZITADEL_CLIENT_ID,
+        project_id=ZITADEL_PROJECT_ID,
+        allowed_scopes={"openid": "OpenID scope", "profile": "Profile scope"},
+    )
+
+
+@pytest.fixture(autouse=True)
+async def reset_openid_cache():
+    """
+    Reset the OpenID configuration cache before each test
+    
+    This fixture automatically runs before each test to ensure
+    a clean state for OpenID configuration caching.
+    """
+    # Note: This assumes ZitadelAuth has a cache that can be reset
```

Comment:
this should call the `openid_config.reset_cache()` method: https://github.com/cleanenergyexchange/fastapi-zitadel-auth/blob/c4a11bc318bf84bf93c1d089377b2bd29905bc6d/src/fastapi_zitadel_auth/openid_config.py#L76-L80

Reply:
do we even need this fixture? Shall I just remove it?